package.json

```diff
@@ -25,7 +25,7 @@
   "scripts": {
     "lint": "npm-run-all --continue-on-error --aggregate-output --parallel lint:*",
     "lint:docs": "markdownlint \"**/*.md\"",
-    "lint:js": "eslint --report-unused-disable-directives .",
```

Comment:
ESLint v9 warns on unused disable directives by default, but we should still use this to tell it to error on them.

Reply:
it's already configured in `eslint-config-eslint`: https://github.com/eslint/eslint/blob/4d11e567baff575146fd267b3765ab2c788aa1e5/packages/eslint-config-eslint/base.js#L383

Reply:
Got it.